docs/cookbook.rst

```diff
@@ -249,3 +249,33 @@ Unless of course something bad happens inside of ``PerformA`` in which case the
     AssertionError:
     Expected: <True>
          but: was <False>
+
+
+Using Either
+===========
+
+Sometimes we need to try one thing then followed by a different thing when the first one fails.
+You could just add a try/except in your code but that breaks up the screenplay pattern.
+So there is an action for that; :func:`~screenpy.actions.Either`::
+
+    the_actor.will(Either(DoAction()).or_(DoDifferentAction())
+
+Screenpy will attempt to perform the first action but if an `AssertionError` is raised
+screenpy will move on to attempt performing the second action instead.  Note that we only catch
+`AssertionError` here allowing for other exceptions to still be raised.
+
+The action also allows users to pass in multiple actions similar to how actors can perform
```

Comment:
Also, do we want to mention that this Action can totally be used as an if/else by using `See`? That seems like a neat thing.
```python
the_actor.will(
    Either(
         See.the(Element(SIGN_UP_CHECKBOX), IsChecked()),  # wow we don't have an IsChecked huh
         and(Uncheck.the(SIGN_UP_CHECKBOX)),  # wow we don't have Check/Uncheck??
    ).or_(
        Log(f"The {SIGN_UP_CHECKBOX} was already checked.")
    ),
    ...
)
```

I swear i remember writing a `Selected`/`Checked` Resolution, was that a custom thing? 🤔

Reply:
I'm hesitant to mention this because:

- it would be basically describing a property of python.
- If we are successful in crafting an action for if/else, I would imagine we'd want to point them to use that.

Reply:
> Though maybe we should call them "Performables"...?

The folder is called `actions`.  But this does raise an interesting question:
Are Actions == Performables? 

I've been operating under the assertion they are not the same thing.  I always considered Actions are Performables but not all Performables are Actions. This is due to Actions needing to be Describable as well.

Reply:
Well sure, if we make one. We don't currently have one, but this one can be used like one. 

I feel like the cookbook is exactly the place for neat tricks like this, no?

Reply:
> Are Actions == Performables? I've been operating under the assertion they are not the same thing.

They are not, you're right. But the things that `Either` can perform are Performables, be they Actions or Tasks.

Reply:
I thought it needed actions because of the use of `describe`

Reply:
Ahh, my mistake. `Either` can perform Performables _and_ it describes the Describables along with  Non-Describables.

Reply:
Now I'm pondering if we technically need a Performable that isn't Describable.  If not, perhaps we should consider combining them into a new Protocol `Action`